point/chain.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "point",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Point",
+  "chain_id": "point_10687-1",
+  "bech32_prefix": "point",
+  "node_home": "$HOME/.pointd",
+  "genesis": {
+    "genesis_url": "https://github.com/pointnetwork/point-chain-config/blob/main/mainnet-1/genesis.json"
+  },
+  "daemon_name": "pointd",
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "apoint",
+        "low_gas_price": 5000000000,
```

Comment:
@DaMandal0rian This is a min gas from ping.pub config. Your number is different. Do we agree to have this number?